### PR TITLE
Corrections to Report

### PR DESCRIPTION
**Pull request by samgdotson**
This PR adjusts the technology fixed costs and the power densities for each scenario and adds corrections to the report.

### CODE REVIEW

**Review by nsryan2 — APPROVED** [review]
LGTM! I'm not sure how this process works (so I won't merge it just yet), but this seems appropriate to me to address the issues you found

File: report/corrections.tex
Comment (npanczyk) [inline report/corrections.tex:33]:
```suggestion
scenario were originally taken from \cite{van_zalk_spatial_2018}, which reports
```

**Review by npanczyk — APPROVED** [review]
Read over all the corrections and literally found a single comma error. Looks great! Thanks for getting this done so quickly.

**Comment by samgdotson** [comment]
@nsryan2 @npanczyk @munkm could you merge this if it looks okay?

**Pull request merged**
